webpack/assets/javascripts/foreman_tools.js

```diff
@@ -80,3 +81,22 @@ export function initTypeAheadSelect(input) {
     width: '400px'
   });
 }
+
+// send get forms through turbolinks
+export function tableUpdated(element) {
+  let uri = new URI(window.location.href);
+  let search;
+
+  if (element !== undefined) {
+    search = $(element).find('.autocomplete-input');
+  } else {
+    search = $('#search');
+  }
+
+  if (search[0] !== undefined) {
```

Comment:
can this condition actually occur? that means there is no search element present on the page